### ISSUE TITLE
memory_limit

### ISSUE THREAD

**Issue by Qrzysio** [DBDiff/DBDiff#69]
In `DBDiff.php` file there is a line which sets up `memory_limit` to `512M`.

`ini_set('memory_limit', '512M');`

This is wrong due to on local machines sometimes we have to set it up even higher. This value should  only be set up by user and not appear in the class.

**Comment by makopov** [comment]
I'm guessing this is so that we don't have to set our systems higher for just this script. However I ran into the same issue where it was not enough and I had to set it to 1024. They should just set this to 1024 in their script. Its not something ran often so its probably okay.

**Comment by aurora** [comment]
Why even set it to a hard-coded value? Should at least be changeable using an option-switch.

**Comment by dataf3l** [comment]
I agree, this also bit me, I had to go fishing for the memory setting.

I guess nobody cares about this project...

**Comment by dataf3l** [comment]
Maybe the program shouldn't put all the records in memory, no?
